rootfs-builder/debian/config.sh

```diff
@@ -8,6 +8,8 @@ OS_VERSION=${OS_VERSION:-9.5}
 # Set OS_NAME to the desired debian "codename"
 OS_NAME=${OS_NAME:-"stretch"}
 
+PACKAGES="util-linux"
```

Comment:
You dont need to add the package here, debian is reusing the config from ubuntu.